src/lib/serviceRoutinesV2/badVerbAllNotDelete.cpp

```diff
@@ -55,11 +56,16 @@ std::string badVerbAllNotDelete
   OrionError   oe(SccBadVerb, ERROR_DESC_BAD_VERB);
 
   ciP->httpHeader.push_back("Allow");
-  ciP->httpHeaderValue.push_back("GET, PATCH, POST, PUT");
+  std::string headerValue = "GET, PATCH, POST, PUT";
```

Comment:
Just for curiosity, is there any 'established' order for the verbs/methods ?

Reply:
Interesting question... what does the CORS specification say about that?

Reply:
CORS specification does not provide any rules for the order of the methods and the example scenarios provided do not seem to be following any pattern.

Please see the example scneario at the end of the subsection: [7.1.5 Cross-Origin Request with Preflight](https://www.w3.org/TR/cors/#cross-origin-request-with-preflight-0).

Reply:
So, NTC (alphabetic order seems ok to me)

I think I used an (by myself invented) "estimated use-frequency order", as: GOT. POST, PUT. PATCH ...
We should change the badVerb service routines to do alphabetic order instead . Can't go wrong ... (except perhaps for 100% backward compatibility :-( )
To be discussed, and nothing to be done in this PR. of course.

Reply:
> We should change the badVerb service routines to do alphabetic order instead . Can't go wrong ... (except perhaps for 100% backward compatibility :-( )
> To be discussed, and nothing to be done in this PR. of course.

Taking into account order doesn't matter from specification point of view, I don't see any value on changing that.

Reply:
> Taking into account order doesn't matter from specification point of view, I don't see any value on changing that.

Uniformity seems like reason enough to me (surprised to hear that you don't care :-))
If it wasn't for the possible backward compatibility I would change it, it's easy enough.
But, let's not